lld/test/ELF/aarch64-build-attributes-be.s

```diff
@@ -0,0 +1,50 @@
+// REQUIRES: aarch64
+// RUN: llvm-mc -triple=aarch64_be %s -filetype=obj -o %t.o
+// RUN: ld.lld %t.o --shared -o %t.so
+// RUN: llvm-readelf -n %t.so | FileCheck %s --check-prefix=NOTE
+
+/// The Build attributes section appearing in the output of
+/// llvm-mc should not appear in the output of lld, because
+/// AArch64 build attributes are being transformed into .gnu.properties.
+
+// Test mc -> big endian, lld -> little endian
```

Comment:
`/// ` for non-RUN non-CHECK comments to make them stand out. (convention in lld tests)